From alan@linux.intel.com  Tue Sep 21 13:29:35 2010
From: Hao Wu <hao.wu@intel.com>
Subject: USB OTG Langwell: Update OTG Kconfig and driver version.
To: greg@kroah.com, randy.dunlap@oracle.com, linux-usb@vger.kernel.org
Date: Thu, 09 Sep 2010 22:35:54 +0100
Message-ID: <20100909213546.12453.19125.stgit@localhost.localdomain>

From: Hao Wu <hao.wu@intel.com>

This patch updated Kconfig for langwell otg transceiver driver.
Add ipc driver(INTEL_SCU_IPC) as a dependency. Driver version is
updated too.

Signed-off-by: Hao Wu <hao.wu@intel.com>
Signed-off-by: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/otg/Kconfig          |    2 +-
 drivers/usb/otg/langwell_otg.c   |    4 ++--
 include/linux/usb/langwell_otg.h |    2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/usb/otg/Kconfig
+++ b/drivers/usb/otg/Kconfig
@@ -69,7 +69,7 @@ config NOP_USB_XCEIV
 
 config USB_LANGWELL_OTG
 	tristate "Intel Langwell USB OTG dual-role support"
-	depends on USB && X86_MRST
+	depends on USB && PCI && INTEL_SCU_IPC
 	select USB_OTG
 	select USB_OTG_UTILS
 	help
--- a/drivers/usb/otg/langwell_otg.c
+++ b/drivers/usb/otg/langwell_otg.c
@@ -1,6 +1,6 @@
 /*
  * Intel Langwell USB OTG transceiver driver
- * Copyright (C) 2008 - 2009, Intel Corporation.
+ * Copyright (C) 2008 - 2010, Intel Corporation.
  *
  * This program is free software; you can redistribute it and/or modify it
  * under the terms and conditions of the GNU General Public License,
@@ -40,7 +40,7 @@
 #include <linux/usb/langwell_otg.h>
 
 #define	DRIVER_DESC		"Intel Langwell USB OTG transceiver driver"
-#define	DRIVER_VERSION		"3.0.0.32L.0003"
+#define	DRIVER_VERSION		"July 10, 2010"
 
 MODULE_DESCRIPTION(DRIVER_DESC);
 MODULE_AUTHOR("Henry Yuan <hang.yuan@intel.com>, Hao Wu <hao.wu@intel.com>");
--- a/include/linux/usb/langwell_otg.h
+++ b/include/linux/usb/langwell_otg.h
@@ -1,6 +1,6 @@
 /*
  * Intel Langwell USB OTG transceiver driver
- * Copyright (C) 2008, Intel Corporation.
+ * Copyright (C) 2008 - 2010, Intel Corporation.
  *
  * This program is free software; you can redistribute it and/or modify it
  * under the terms and conditions of the GNU General Public License,
